src/panels/ExoplanetsPanel/ExoplanetEntry.tsx

```diff
@@ -9,21 +10,24 @@ interface ExoplanetProps {
 }
 
 export function ExoplanetEntry({ name, isAdded, onClick }: ExoplanetProps) {
+  const { t } = useTranslation('panel-exoplanets', { keyPrefix: 'exoplanet-entry' });
+
   return (
     <Group grow mb={'xs'}>
-      {
-        <ActionIcon
-          size={'lg'} // Fixed size, adjust as needed
-          key={`${name}button`}
-          variant={isAdded ? 'outline' : 'filled'}
-          style={{ flexGrow: 0 }}
-          onClick={onClick}
-          color={isAdded ? 'red' : 'blue'}
-          aria-label={`${isAdded ? 'Remove' : 'Add'} system: ${name}`}
-        >
-          {isAdded ? <MinusIcon /> : <PlusIcon />}
-        </ActionIcon>
-      }
+      <ActionIcon
+        size={'lg'} // Fixed size, adjust as needed
+        key={`${name}button`}
+        variant={isAdded ? 'outline' : 'filled'}
+        style={{ flexGrow: 0 }}
+        onClick={onClick}
+        color={isAdded ? 'red' : 'blue'}
+        aria-label={`${
+          isAdded ? t('remove-system-aria-label') : t('add-system-aria-label')
+        }: ${name}`}
```

Comment:
Should this use the name as a variable in the string? 

Or, based on our discussion, this should probably be avoided in general when possible. But that means removing the anchor node at the end of this stinrg in the geolocation panel as well: 

```
"no-anchor-panel": "Currently there is no data for locations on {{anchor}}",
```

Reply:
Yeah I'm not sure which one I prefer anymore.. if I was to translate the "no-anchor-panel" and it only said `"Currently there is no data for locations on"` I would be confused, since the translation might be different depending on what comes after "on"? 

But in this case it was to simplify the translation and to not have the semicolon be part of the translation and accidently be missed between languages. Though since its an aria label it doesn't matter as much?

Reply:
Ah, good points. Feel free to ignore this comment!

Thanks for thorough answers :)